### PR TITLE
validate.py: "one" tag check (in tandem with neaten.py)

### PR DESCRIPTION
**Pull request by nschneid**
cf. UniversalDependencies/UD_English-EWT@3182a98

### CODE REVIEW

**Review by amir-zeldes — CHANGES_REQUESTED** [review]

File: _build/utils/validate.py [hunk 1]
```diff
@@ -942,7 +945,8 @@ def flag_dep_warnings(id, tok, pos, lemma, func, parent, parent_lemma, parent_id
 						print("WARN: q root may not have wh child " + wh + inname)
 
 	suspicious_pos_tok = [("*","DT","only","RB"),
-						  ("*","JJ","one","CD")]
+			      ("*","JJ","one","CD"),    # may be redundant with new amod-dependent-of-'one' check
+			      ("no","DT","one","CD")]
```
[the remark this answers is not in the document]
Comment (nschneid) [reply _build/utils/validate.py:949]: Ah, that case doesn't come up in EWT.

[the remark this answers is not in the document]
Comment (amir-zeldes) [reply _build/utils/validate.py:949]: OK, but I think it's also not needed given that you have the check above. If you remove this one I'll merge it for GUM as well.

[the remark this answers is not in the document]
Comment (nschneid) [reply _build/utils/validate.py:949]: The point was that it should be no/DT one/NN

[the remark this answers is not in the document]
Comment (amir-zeldes) [reply _build/utils/validate.py:949]: Right, but not in "no/DT one/CD group" - there it's a regular numerical one, and the DT is attaching to "group". The first check should still work and will not flag this case incorrectly, right?

[the remark this answers is not in the document]
Comment (nschneid) [reply _build/utils/validate.py:949]: Oh I see it now. Yes. "no one" is a special case of det(one/NN, *). So lines 948 & 949 are unnecessary.

[the remark this answers is not in the document]
Comment (amir-zeldes) [reply _build/utils/validate.py:949]: Great, if you revise the PR I'll merge this.

**Comment by nschneid** [comment]
Done. You may also want to run neaten.py on the generated GUM .conllus; it now checks a bunch of features etc.

**Comment by amir-zeldes** [comment]
Ideally we should integrate it into the GUM buildbot, but I see there is now a lot of redundancy between the buildbot and neaten... Something to look into in the future I guess.

**Pull request merged**